src/rabbit_control_main.erl

```diff
@@ -748,12 +757,24 @@ ensure_app_running(Node) ->
         Other -> Other
     end.
 
+notify_if_timeout(Pid, Ref, Timeout) ->
+    timer:send_after(Timeout, Pid, {Ref, {timeout, Timeout}}).
+
 call(Node, {Mod, Fun, Args}) ->
     rpc_call(Node, Mod, Fun, lists:map(fun list_to_binary_utf8/1, Args)).
 
 call(Node, {Mod, Fun, Args}, Timeout) ->
```

Comment:
Compiler warning:

src/rabbit_control_main.erl:766: Warning: function call/3 is unused